PokeAlarm/Events/MonEvent.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, data):
 
         # Time Left
         self.disappear_time = datetime.utcfromtimestamp(data['disappear_time'])
+        self.spawn_time_left = get_time_in_seconds(self.disappear_time)
```

Comment:
What do you think about changing all the event attributes to just 'time_left' so there is some consistency between them?

Reply:
👌